pkg/tools/renderer/colors.go

```
@@ -0,0 +1,547 @@
+		}
+		return "✘"
+	}
+
+	if isKEV {
+		return c.Colorize("🚨 KEV LISTED", Bold+BrightRed)
+	}
+	return c.Colorize("✘", c.Error)
+}
+
+// colorRiskLevels colors risk level indicators in the text
+func (c *ColorConfig) colorRiskLevels(text string) string {
```

golangci-lint (1.64.8)

func `(*ColorConfig).colorRiskLevels` is unused

(unused)